clustered/integration-test/src/test/java/org/ehcache/clustered/replication/BasicClusteredCacheOpsReplicationMultiThreadedTest.java

```diff
@@ -241,6 +242,7 @@ public void testBulkOps() throws Exception {
   }
 
   @Test
+  @Ignore
```

Comment:
Is this being ignored with expectation for a future repair?  If so, an issue number should be mentioned.